stem/boundary.py

```diff
@@ -0,0 +1,130 @@
+from typing import List, Dict, Any, Union, Optional
+from dataclasses import dataclass, field
+from abc import ABC, abstractmethod
+
+
+@dataclass
+class BoundaryParametersABC(ABC):
+    """
+    Abstract base class for boundary parameters
+    """
+
+    pass
+
+    @property
+    @abstractmethod
+    def is_constraint(self) -> bool:
+        """
+        Property which indicates if boundary condition is a constraint.
+        Returns:
+            bool
+        """
+        pass
+
+
+@dataclass
+class DisplacementConstraint(BoundaryParametersABC):
+    """
+    Class containing the boundary parameters for displacement constraint
+
+    Inheritance:
+        - :class:`BoundaryParametersABC`
+
+    Attributes:
+        active (List[bool]): Activate/deactivate constraint for each direction.
+        is_fixed (List[bool]): Specify if constraint is fixed for each direction.
+        value (List[float]): Displacement constraint for direction [m].
```

Comment:
```suggestion
        - active (List[bool]): Activate/deactivate constraint for each direction.
        - is_fixed (List[bool]): Specify if constraint is fixed for each direction.
        - value (List[float]): Displacement constraint for direction [m].
```

please correct all docstrings in the PR with the new docstring style, such that readthedocs will be generated nicely